src/e-show.js

```diff
@@ -154,29 +163,23 @@ program
 
 program
   .command('out')
-  .description('outdir name, e.g. "Testing"')
-  .action(() => {
-    try {
-      console.log(evmConfig.current().gen.out);
-    } catch (e) {
-      fatal(e);
-    }
-  });
-
-program
-  .command('outdir')
-  .description('outdir path, e.g. "/$root/src/out/Testing"`')
+  .description('Show outdir name, e.g. "Testing"')
+  .option('--path', 'Output as JSON')
```

Comment:
I think it's more intuitive to make showing the `outdir` path an option under `out` 🤔